qfieldsync/gui/map_layer_config_widget.py

```diff
@@ -74,14 +75,23 @@ def __init__(self, layer, canvas, parent):
         self.isGeometryLockedCheckBox.setEnabled(self.layer_source.can_lock_geometry)
         self.isGeometryLockedCheckBox.setChecked(self.layer_source.is_geometry_locked)
 
-        self.photoNamingTable = PhotoNamingTableWidget()
-        self.photoNamingTable.addLayerFields(self.layer_source)
-        self.photoNamingTable.setLayerColumnHidden(True)
+        self.attachmentNamingTable = AttachmentNamingTableWidget()
+        self.attachmentNamingTable.addLayerFields(self.layer_source)
+        self.attachmentNamingTable.setLayerColumnHidden(True)
 
-        # append the photo naming table to the layout
+        # append the attachment naming table to the layout
         if layer.type() == QgsMapLayer.VectorLayer:
-            self.layout().insertRow(-1, self.tr("Photo naming"), self.photoNamingTable)
-            self.photoNamingTable.setEnabled(self.photoNamingTable.rowCount() > 0)
+            self.layout().insertRow(
+                -1, self.tr("Attachment\nnaming"), self.attachmentNamingTable
```

Comment:
Why don't you `QLabel.setWordWrap(true);`, instead of forcing newline? Or QT is not smart enough and makes the first column too wide?

Reply:
Not smart enough